src/v/kafka/group_probe.h

```diff
@@ -46,9 +54,87 @@ class group_offset_probe {
             labels)});
     }
 
+    void setup_public_metrics(
+      const kafka::group_id& group_id, const model::topic_partition& tp) {
+        namespace sm = ss::metrics;
+
+        if (config::shard_local_cfg().disable_public_metrics()) {
+            return;
+        }
+
+        auto group_label = sm::label("group");
+        auto topic_label = sm::label("topic");
+        auto partition_label = sm::label("partition");
+        std::vector<sm::label_instance> labels{
+          group_label(group_id()),
+          topic_label(tp.topic()),
+          partition_label(tp.partition())};
+
+        _public_metrics.add_group(
+          prometheus_sanitize::metrics_name("kafka:consumer:group"),
+          {sm::make_gauge(
+             "committed_offset",
+             [this] { return _offset; },
+             sm::description("Consumer group committed offset"),
+             labels)
+             .aggregate({sm::shard_label})});
```

Comment:
Is this metric reported from one shard only per group (i.e. is the  kafka::group managed by one shard only)? The answer is probably yes, but otherwise summing the committed offset doesn't make sense.

Reply:
> Is this metric reported from one shard only per group (i.e. is the kafka::group managed by one shard only)? The answer is probably yes, but otherwise summing the committed offset doesn't make sense.

As far as I can tell, yes the group is managed per shard. The list of groups is maintained by `kafka::group_manager` and the `kafka::group_manager` is sharded within redpanda.
```
    ss::sharded<kafka::group_manager> _group_manager;
    ss::sharded<kafka::group_manager> _co_group_manager;
```

Reply:
So let's keep the aggregation

Reply:
It's just a funny spelling of "Reporting which shard this group is hosted on is not relevant"